src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -1153,6 +1154,70 @@ private void BuildCompiledBindings(IndentedStringBuilder writer)
 						writer.AppendLineIndented($"Owner = owner;");
 					}
 
+					using (writer.BlockInvariant($"void {bindingsInterfaceName}.NotifyXLoad(string name)"))
```

Comment:
Could you add a sourcegen test that shows the generated code for that scenario??

Reply:
Yup sure. Will add one now.